selfhost/typechecker.jakt

```diff
@@ -889,7 +890,7 @@ struct CheckedCall {
     callee_throws: bool
 }
 
-function unknown_type_id() -> TypeId => TypeId(module: ModuleId(id: 0), id: 0)
+function unknown_type_id() -> TypeId => TypeId(module: ModuleId(id: 0), id: 16)
```

Comment:
is it possible to say `TypeId(module: ModuleId(id: 0), id: BuiltinType::Unknown)` here so we don't have to remember that it's `16`?

Reply:
Yeah I think it's possible. Let me try.